downstreamadapter/eventcollector/event_collector.go

```diff
@@ -232,6 +244,14 @@ func (c *EventCollector) processFeedback(ctx context.Context) {
 		case <-ctx.Done():
 			return
 		case feedback := <-c.ds.Feedback():
+
+			if feedback.IsAreaFeedback() {
```

Comment:
What these codes mean ?

Reply:
Area represent a changefeed in eventCollector's DS. 
AreaFeedBack means this feedback is a area level feedback, it will pause or resume an area(changefeed).